### ISSUE TITLE
Notifications and Week Ahead

### ISSUE THREAD

**Issue by sakroyd** [b0mbays/stockport-bin-collections#1]
Hey,

This is an awesome integration and is really useful. Not sure whether this is possible, but is it possible to set an alert in the morning should that be the day that a collection is due? And secondly, is it possible for the integration to bring in the 'week ahead' of bins? i.e. if it knows todays date - if it could bring in that day +5 ?

Thanks for all your work either way, its awesome

**Comment by b0mbays** [comment]
Hey @sakroyd good to see someone else using this, I thought it was just me 😄 

I can look into setting an alert. Would that be a push notification to your phone from HA on the day that you think would be best?

Regarding the following week, this would definitely be possible as the website gives you the next weeks collections too. I'll get this implemented as a separate entity to make things easier.

**Comment by sakroyd** [comment]
Yeah, I think a push notification from HA would be awesome, and yeah a second entity maybe with the week if that is published by Stockport Council - look forward to it!
